src/components/layout/footer.tsx

```diff
@@ -0,0 +1,113 @@
+"use client";
+
+import copyrightLogo from "@public/images/icons/copyright.svg";
+import githubIcon from "@public/images/icons/github.svg";
+import linkedinIcon from "@public/images/icons/linkedin.svg";
+import slackIcon from "@public/images/icons/slack.svg";
+import Image from "next/image";
+import React, { CSSProperties, useEffect, useState } from "react";
+
+const Footer = React.memo(() => {
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobile(window.innerWidth <= 768);
+    };
+
+    window.addEventListener("resize", handleResize);
+    handleResize();
+
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
+  const containerStyle: CSSProperties = {
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    flexWrap: isMobile ? "wrap" : "nowrap",
+    textAlign: isMobile ? "center" : "left",
+  };
+
+  const leftDivStyle = {
+    marginLeft: isMobile ? "0" : "60px",
+    marginTop: isMobile ? "10px" : "20px",
+  };
```

Comment:
Missing type if you are following other styles.

Reply:
Updated to use tailwind classes, let me know what you think